addons/web/static/tests/webclient/settings_form_view/settings_form_view.test.js

```diff
@@ -2061,3 +2222,102 @@ test("Open settings from url, with setting id anchor", async () => {
     await runAllTimers();
     expect(".o_setting_highlight").toHaveCount(0);
 });
+
+test("swipe settings in mobile [REQUIRE TOUCHEVENT]", async () => {
+    mockTouch(true);
+    mockService("ui", (env) => {
+        Object.defineProperty(env, "isSmall", {
+            value: true,
+        });
+        return {
+            bus: new EventBus(),
+            get size() {
+                return SIZES.XS;
+            },
+            get isSmall() {
+                return true;
+            },
+        };
+    });
```

Comment:
Does this mock is really required ? I guess the goal is the `size == XS` but, is it still useful ?